Installation instructions incorrect

In a docker container "node:20" I followed the README:

`npx @netwerk-digitaal-erfgoed/ld-workbench -- --init`

which suggested to run `npx ld-workbench --init`:

`Configuration directory/file ./pipelines/ could not be found.
If this is your first run of LDWorkbench, you might want to use `npx ld-workbench --init` to setup an example workbench project.` 

I did that, but that resulted in

`npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@netwerkdigitaalergoed%2fld-workbench - Not found
npm ERR! 404 
npm ERR! 404  '@netwerkdigitaalergoed/ld-workbench@*' is not in this registry.`

--- a/.npmignore
+++ b/.npmignore
@@ -4,6 +4,8 @@
 .vscode
 .idea
 src/
-static/
+static/ao
+static/figures
+static/tests
 tsconfig.json
 .eslintrc.json

--- a/README.md
+++ b/README.md
@@ -84,7 +84,7 @@ stages:
 2. Install LD Workbench:
 
    ```sh
-   npx @netwerk-digitaal-erfgoed/ld-workbench -- --init
+   npx @netwerk-digitaal-erfgoed/ld-workbench --init
    ```
 
    Your workbench is now ready for use.

--- a/src/utils/init.ts
+++ b/src/utils/init.ts
@@ -5,13 +5,11 @@ import glob from 'glob'
 
 export default function init(): void {
   const $dirname = path.dirname(fileURLToPath(import.meta.url));
-  if(fs.existsSync('pipelines')) {
-    throw new Error('The --init script found an existing directory "pipelines". Make sure this directory does not exists before running this script.')
+  if(fs.existsSync(path.join('pipelines', 'configurations', 'example'))) {
+    throw new Error('The --init script found an existing directory "' + path.join('pipelines', 'configurations', 'example') + '". Make sure this directory does not exists before running this script.')
   }
-  fs.mkdirSync('pipelines')
-  fs.mkdirSync(path.join('pipelines', 'data'))
-  fs.mkdirSync(path.join('pipelines', 'configurations'))
-  fs.mkdirSync(path.join('pipelines', 'configurations', 'example'))
+  fs.mkdirSync(path.join('pipelines', 'data'), {recursive: true})
+  fs.mkdirSync(path.join('pipelines', 'configurations', 'example'), {recursive: true})
   const filepaths = glob.sync(path.join($dirname, '..', '..', 'static', 'example', '*'))
   for(const filepath of filepaths) {
     fs.copyFileSync(filepath, path.join('pipelines', 'configurations', 'example', path.basename(filepath)))

